generator/App.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+	<appSettings>
+		<add key="MIN_TEMPERATURE" value="0" />
+		<add key="MAX_TEMPERATURE" value="100" />
+		<add key="HOW_MANY_DATA_PER_MINUTE_FOR_TEMPERATURE" value="12" />
```

Comment:
WDYT about smaller value? It seems a little bit talkative for me, 3-5 times per minute?